html/index.html

```diff
@@ -6,9 +6,10 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="../css/global.css" />
     <link rel="stylesheet" href="../css/style.css" />
-    <title>Document</title>
+    <title>homepage</title>
   </head>
   <body>
     <h1>Welcome To LM Techub!</h1>
+    <p>This is a silly line</p>
```

Comment:
I dont get what you are doing. can you more seriouser

Reply:
what do you mean by seriouser?